config/initializers/hmrc_interface.rb

```diff
@@ -0,0 +1,8 @@
+require 'hmrc_interface'
+
+HmrcInterface.configure do |config|
```

Comment:
If you want 'proper' capitalisation, you could add HMRC to the inflections

Reply:
yeah, i decided against it for now as `HMRCInterface` does not feel right and was thinking to not namespace as `HMRC::Interface` but rather rely on it being just the "name" of the API and use standard CamelCase. may change in the future. what do you think?